ajax/libs/react-block-ui/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "react-block-ui",
+  "filename": "reactblockui.min.js",
+  "description": "Block UI for react",
+  "homepage": "https://availity.github.io/react-block-ui/",
+  "keywords": [
+    "availity",
+    "loader",
+    "blockui",
+    "redux",
+    "react",
+    "component",
+    "components",
+    "react-component",
+    "ui"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+ssh://git@github.com/Availity/react-block-ui.git"
+  },
+  "license": "MIT",
+  "author": "Evan Sharp <evan.sharp@availity.com> (https://github.com/TheSharpieOne)",
+  "npmName": "react-block-ui",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "**/+(reactblockui.!(cjs|es)|style)*+(css|js|map)"
```

Comment:
Let's modify this to `**/+(reactblockui.!(cjs|es)|style)*.+(css|js|map)` which add a `.` between `*` and +(css|js|map). Though the results are the same but the latter one make more sense. Thank you.